sparkle/src/components/DataTable.tsx

```diff
@@ -80,18 +79,20 @@ export function DataTable<TData extends TBaseData>({
   widthClassName = "s-w-full s-max-w-4xl",
   filter,
   filterColumn,
-  initialColumnOrder,
   columnsBreakpoints = {},
   pagination,
   setPagination,
+  sorting,
+  setSorting,
 }: DataTableProps<TData>) {
   const windowSize = useWindowSize();
-  const [sorting, setSorting] = useState<SortingState>(
-    initialColumnOrder ?? []
-  );
+
   const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([]);
 
   const isServerSidePagination = !!totalRowCount && totalRowCount > data.length;
+  const isServerSideSorting = !!sorting && !!setSorting;
```

Comment:
not sure about that one, sorting/setSorting in useTable are needed for client or server sort - in all cases we need to keep the state of the sort somewhere